guides/common/modules/proc_running-salt.adoc

```diff
@@ -1,43 +1,40 @@
 [id="Running_Salt_{context}"]
 = Running Salt
 
-You can run arbitrary Salt functions using remote execution.
-
-Most commonly, you can run `salt.highstate` on one or more Salt Minions.
-This applies all relevant Salt States on your managed hosts.
-
-.Procedure
-. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
-. Select one or multiple hosts.
-. Click *Run Salt* from the _actions_ drop down menu.
+You can run arbitrary Salt functions using remote execution, for example `salt.highstate`, on one or more Salt Minions.
+This applies all relevant Salt States on your managed hosts and includes an optional test mode that tells you what would happen without actually changing anything.
```

Comment:
What test mode do you refer to? I might have overseen it in the wizard, but I can't see such test mode.

Reply:
I haven't heard about it and since it hasn't been part of the docs before, I would suggest to drop it.

Reply:
Applied.